frontend/app/templates/work_packages/tabs/_revision_activity.html

```diff
@@ -0,0 +1,22 @@
+<div class="work-package-details-activities-activity-contents">
+  <div class="comments-number">
+    <span ui-sref="work-packages.list.details.activity.details({ activity_no: activityNo })">
+      <a name="{{ activityNo }}"
+         ng-bind="'#' + activityNo"
+         href="#{{ activityNo }}"></a>
+    </span>
+  </div>
+  <img class="avatar" ng-src="{{ userAvatar }}" alt="Avatar" title="{{userName}}" ng-if="userAvatar" />
```

Comment:
Thanks. I'll take those comments into account when moving this into its own templates, as your comments currently apply to [_user_activity.html](https://github.com/opf/openproject/blob/feature/repository_activity/frontend/app/templates/work_packages/tabs/_user_activity.html) as well (cf. todo in PR note)